buildkite/test_plans_test.go

```diff
@@ -0,0 +1,98 @@
+package buildkite
+
+import (
+	"fmt"
+	"net/http"
+	"reflect"
+	"testing"
+)
+
+func TestTestPlansService_Get(t *testing.T) {
+	setup()
+	defer teardown()
+
+	mux.HandleFunc("/v2/analytics/organizations/my-great-org/suites/suite-1/test_plan", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
```

Comment:
`testMethod` being in a different file feels hard to eye trace. Not a you change, just noticed it.

Reply:
Yeah it threw me a little at first. But then I guess `mux` is also in a different file so I assumed things were just coming from somewhere else!